Remotion/ObjectBinding/Web.IntegrationTests/BocAutoCompleteReferenceValue/ReadOnlyBocAutoCompleteReferenceValueTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests;
+using Remotion.Web.Development.WebTesting.ControlObjects;
+using Remotion.Web.Development.WebTesting.ControlObjects.Selectors;
+using Remotion.Web.Development.WebTesting.ControlSelection;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+using Remotion.Web.Development.WebTesting.FluentControlSelection;
+
+namespace Remotion.ObjectBinding.Web.IntegrationTests.BocAutoCompleteReferenceValue
+{
+  [TestFixture]
+  public class ReadOnlyBocAutoCompleteReferenceValueTest : IntegrationTest
+  {
+    [Test]
+    public void Validate_OnReadOnlyControl_CreatesValidationErrors ()
+    {
+      var home = Start();
+      var webButtonControlObject = home.GetControl(new LocalIDControlSelectionCommand<WebButtonControlObject>(new WebButtonSelector(), "ValidateButton"));
```

Comment:
this should work with `home.WebButtons().GetByLocalID("ValidateButton")` Please verify and update.

And just info for how other tests are written:
most of the time, we suffix the `.Click()` into the call since there's not need to make a local variable for it.